dsgrid/dataset/dataset_schema_handler_standard.py

```diff
@@ -95,16 +98,17 @@ def make_project_dataframe(self, project_config, scratch_dir_context: ScratchDir
         if SCALING_FACTOR_COLUMN in ld_df.columns:
             ld_df = apply_scaling_factor(ld_df, {VALUE_COLUMN})
         ld_df = self._apply_fraction(ld_df, {VALUE_COLUMN})
-        project_metric_records = project_config.get_base_dimension(
-            DimensionType.METRIC
-        ).get_records_dataframe()
-        ld_df = self._convert_units(ld_df, project_metric_records, {VALUE_COLUMN})
+        project_metric_records = self._get_project_metric_records(project_config)
+        ld_df = self._convert_units(ld_df, project_metric_records)
         ld_df = self._convert_time_dimension(
             ld_df, project_config, VALUE_COLUMN, scratch_dir_context
         )
+        # TODO DT: this has a bug...does not account for duplicates
```

Comment:
This needs to be fixed...not yet sure about this PR.

Reply:
This is now fixed. It was likely not a serious issue and probably only affected dGen. Some subsectors did not exist in some census tracts and so had null id values in load_data_lookup. When the aggregation of census tracts to counties occurred, the result was that the same combination of county and other stacked dimensions had both valid timestamps and values and rows with null timestamps and values. This would not have affected aggregations or totals.